### PR TITLE
[0.9.x] KOGITO-1644 Move docker tests to failsafe

### PR DESCRIPTION
**Pull request by jstastny-cz**
* KOGITO-1644 Move docker tests to failsafe

cherry-pick of #198

### CODE REVIEW

File: process-usertasks-with-security-oidc-springboot/pom.xml [hunk 1]
```diff
@@ -13,6 +13,11 @@
   <name>Kogito Examples :: Process Usertasks With Security OIDC Spring Boot</name>
   <description>Kogito usertasks orchestration with security enabled on REST api - Spring Boot</description>
 
+  <properties>
+    <container.image.keycloak>quay.io/keycloak/keycloak:${container.image.keycloak.version}</container.image.keycloak>
```
Comment (cristianonicolai) [inline process-usertasks-with-security-oidc-springboot/pom.xml:17]: perhaps move this to root pom so it can be reused?

Comment (jstastny-cz) [reply process-usertasks-with-security-oidc-springboot/pom.xml:17]: hm, that's what is in master atm .... I can do it master in a separate PR.

Comment (jstastny-cz) [reply process-usertasks-with-security-oidc-springboot/pom.xml:17]: I mean in this PR it's not needed. If there are other examples to be added later, the PR adding the new ones will refactor this (but probably not on 0.9.x).

Comment (evacchi) [reply process-usertasks-with-security-oidc-springboot/pom.xml:17]: if this is not strictly required for 0.9.x let's keep this for master only

**Review by evacchi — APPROVED** [review]

File: process-usertasks-with-security-oidc-springboot/pom.xml (hunk 1 above)
Comment (jstastny-cz) [reply process-usertasks-with-security-oidc-springboot/pom.xml:17]: Tracking the proposal as KOGITO-1867 and targeting 0.10.0

**Pull request merged**